packages/helper/fit_and_predict.py

```diff
@@ -63,10 +65,10 @@ async def fit_and_predict(metric_name,start_time,end_time,url,prom_query,write_b
                 old_model = model_from_json(fin.read())  # Load model
                 
             logger("Retraining ML model","warning")
-            model = Prophet(seasonality_mode='multiplicative').fit(df,init=stan_init(old_model))
+            model = Prophet(changepoint_prior_scale=params["changepoint_prior_scale"],seasonality_prior_scale=params["seasonality_prior_scale"],holidays_prior_scale=params["holidays_prior_scale"],changepoint_range=params["changepoint_range"],seasonality_mode=params["seasonality_mode"],).fit(df,init=stan_init(old_model))
```

Comment:
@WeepingClown13 make this model var prophet_model instead of model to avoid conflict